specs/src/specs/namespace.md

```diff
@@ -87,6 +87,10 @@ For additional information on the significance and application of the reserved n
 
 Applications MUST refrain from using the [reserved namespaces](#reserved-namespaces) for their blob data.
 
+Celestia does not ensure the prevention of non-reserved namespace collisions. 
+Consequently, two distinct applications might use the same namespace.
+It is the responsibility of these applications to be cautious and manage the implications and consequences arising from such namespace collisions.
```

Comment:
we might also want to link to the "Woods Attack" if this is not already linked somewhere

https://forum.celestia.org/t/woods-attack-on-celestia/59

Reply:
Sure, will do in a separate PR [to not to dismiss the current approvals].